plugins/node/opentelemetry-instrumentation-express/src/enums/ExpressLayerType.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export enum ExpressLayerType {
```

Comment:
I saw you moved this enum but didn't move `enum.ts` for the graphql, i think this should be consistent too. Is the goal to have all enums in `enums/` ?

Reply:
I moved ExpressLayerType because it was the only enum left in enums.ts

The idea of this PR was to stanardise non-official (not in "semantic conventions") attributes, see https://github.com/open-telemetry/opentelemetry-js/issues/2184

But of course moving other (all) enums into a enums/<EnumName>.ts could be beneficial as well

Reply:
I generally would prefer to only move enums that are part of semantic conventions and make a different PR if you want to move other enums (which makes totally sense) to avoid mixing things.
However since its only the express enum i think this can still go through